.github/workflows/ci-wheels.yml

```diff
@@ -32,36 +32,42 @@ jobs:
       fail-fast: false
       matrix:
         # TBD: extend to support windows
-        os: ["ubuntu-latest", "macos-11", "macos-14"]
+        # TODO: enable macos support
+        os: ["ubuntu-latest"] # "macos-latest"
+        arch: ["x86_64", "arm64"]
         include:
         - os: ubuntu-latest
           arch: x86_64
           incdir: /usr/include/udunits2
           libdir: /usr/lib64
           xml_path: /usr/share/udunits/udunits2.xml
-        - os: macos-11
-          arch: x86_64
-          incdir: /usr/local/Cellar/udunits/2.2.28/include
-          libdir: /usr/local/Cellar/udunits/2.2.28/lib
-          xml_path: /usr/local/Cellar/udunits/2.2.28/share/udunits/udunits2-common.xml
-        - os: macos-14
+#        - os: macos-latest
+#          arch: x86_64
+#          incdir: /usr/local/Cellar/udunits/2.2.28/include
+#          libdir: /usr/local/Cellar/udunits/2.2.28/lib
+#          xml_path: /usr/local/Cellar/udunits/2.2.28/share/udunits/udunits2-common.xml
+#        - os: macos-latest
+#          arch: arm64
+#          incdir: /opt/homebrew/Cellar/udunits/2.2.28/include
+#          libdir: /opt/homebrew/Cellar/udunits/2.2.28/lib
+#          xml_path: /opt/homebrew/Cellar/udunits/2.2.28/share/udunits/udunits2-common.xml
+        exclude:
+        - os: ubuntu-latest
           arch: arm64
-          incdir: /opt/homebrew/Cellar/udunits/2.2.28/include
-          libdir: /opt/homebrew/Cellar/udunits/2.2.28/lib
-          xml_path: /opt/homebrew/Cellar/udunits/2.2.28/share/udunits/udunits2-common.xml
 
     steps:
     - uses: actions/checkout@v4
       with:
         fetch-depth: 0
 
     - name: "Building ${{ matrix.os }} (${{ matrix.arch }}) wheels"
-      uses: pypa/cibuildwheel@v2.16.5
+      uses: pypa/cibuildwheel@v2.20.0
       env:
-        CIBW_SKIP: "cp36-* cp37-* cp38-* cp312-* pp* *-musllinux*"
+        CIBW_SKIP: "cp39-* cp313-* pp* *-musllinux*"
```

Comment:
do we need 3.12?

Reply:
yea i like showing that we can build the wheels on 3.12 even if we are not testing it for now

Reply:
yeah cool no drama